composer.json

```diff
@@ -20,8 +20,6 @@
         "doctrine/common": "~2.4",
         "twig/twig": "~1.23|~2.0",
         "psr/log": "~1.0",
-        "symfony/security-acl": "~2.7",
```

Comment:
This was a wrong merge in https://github.com/symfony/symfony/commit/67df429d99c1cf55227f8bb065f88872e5b3edeb#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780
Not sure how we can prevent this from happening again and again.